src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/ConsumerContext.cs

```diff
@@ -10,16 +10,15 @@ namespace MassTransit.KafkaIntegration
     public interface ConsumerContext<TKey, TValue> :
         PipeContext,
         IConsumerLockContext<TKey, TValue>,
-        IAsyncDisposable
+        IDisposable
```

Comment:
Why not just leave it async, since it's consistent, and works with Supervisor, etc. Would also automatically get stopped if added as a consume agent on the client context.